book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader	:rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] there should be a method add_rental i.e 
```
 def add_rental(person, date) 
    Rental.new(date, self, person)
  end
```

Reply:
@VICTOR-CHUKWU thank you very much for your review and the pointing towards the clear solution. I have referenced the Project Requirements a lot and this might be out of my knowledge. Thank you very much for these two methods. I have corrected it as below

```
class Book
  attr_accessor :title, :author
  attr_reader	:rentals

  def initialize(title, author)
    @title = title
    @author = author
    @rentals = []
  end

  def add_rental(person, date)
    Rental.new(date, self, person)
  end
end
```

The above code does not trigger any linter errors. I have a doubt, for `Rental` to work do I need to write a line at the top of the `book.rb` file like require `./rentals.rb` ?? or no need??

Thanks in advance.

Reply:
I have tested without `require ./rentals.rb`, It gave error as expected.
So I used the line at top of both `book.rb` and `person.rb` files.

```
[admin@ANTARCTICA oop-school-library]$ irb
irb(main):001:0> require "./book.rb"
=> true
irb(main):002:0> require "./person.rb"
=> true
irb(main):003:0> book = Book.new("TITLE", "AUTHOR")
=> #<Book:0x0000562958eca220 @author="AUTHOR", @rentals=[], @title="TITLE">
irb(main):004:0> person = Person.new(22, "PERSON")
=> #<Person:0x0000562958c504e0 @age=22, @id=70, @name="PERSON", @parent_permission=true, @rentals=[]>
irb(main):005:0> book.add_rental("22-11-01", person)
=> 
#<Rental:0x0000562958ca86b8              
 @book=#<Book:0x0000562958eca220 @author="AUTHOR", @rentals=[#<Rental:0x0000562958ca86b8 ...>], @title="TITLE">,
 @date="22-11-01",                       
 @person=                                
  #<Person:0x0000562958c504e0            
   @age=22,                              
   @id=70,                               
   @name="PERSON",
   @parent_permission=true,
   @rentals=[#<Rental:0x0000562958ca86b8 ...>]>>
irb(main):006:0> 
```

Thank you so much @VICTOR-CHUKWU